CMakeLists.txt

```diff
@@ -209,6 +210,9 @@ if(APPLE)
         set(CMAKE_BUILD_WITH_INSTALL_RPATH ON CACHE
             BOOL "Do Build Using Library Install RPath" FORCE)
     endif(NOT CMAKE_BUILD_WITH_INSTALL_RPATH)
+    SET(GCC_COVERAGE_COMPILE_FLAGS "-L/opt/local/lib -Qunused-arguments")
```

Comment:
My issues are (1) using the text "/opt/local/lib"; you can find other ways to determine this, and then it becomes, e.g., "${MACPORTS_PREFIX}/lib". (2) I'm worried that adding the "-L${MACPORTS_PREFIX}/lib" at this point in the C*FLAGS will put it before local-to-build paths, such that already-installed libraries will be found before local-to-build ones. You'll want to test this change carefully to make sure my scenario is not happening.